dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -274,15 +277,12 @@ DataInterfaceIIDM::initFromIIDM() {
   }
 
   //===========================
-  //  ADD 3WTFO INTERFACE
+  //  CONVERT 3WTFO
```

Comment:
Maybe writing CONVERT THREE WINDINGS TRANSFORMERS would be clearer ?

Reply:
ok, thanks